contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
```

Comment:
So I will go ahead and add a cloudslang flow to examples action pack.
and when we wil have cloudslang action pack we will rethink it.
@meirwah @Kami  OK?